subsys/partition_manager/Kconfig

```diff
@@ -130,26 +130,66 @@ config PM_SINGLE_IMAGE
 	  The option is always defined in builds where a TF-M image for the
 	  Secure Execution environment is generated.
 
-menuconfig PM_EXTERNAL_FLASH
+config PM_EXTERNAL_FLASH
 	bool "Support external flash in Partition Manager"
+	depends on PM_EXTERNAL_FLASH_DEV_NAME != ""
+	depends on PM_EXTERNAL_FLASH_SIZE != 0
 
-if PM_EXTERNAL_FLASH
+config NRF_DK_QSPI_EXTERNAL_FLASH
```

Comment:
This is a prompt less symbol, but I don't find any places it's being selected ?

Reply:
Its removed now